packages/core/src/index.js.flow

```diff
@@ -62,8 +62,8 @@ declare export function mergeMapConcurrently <A, B> (concurrency: number): (f: (
 declare export function mergeMapConcurrently <A, B> (concurrency: number, f: (A) => Stream<B>): (s: Stream<A>) => Stream<B>
 declare export function mergeMapConcurrently <A, B> (concurrency: number): (f: (A) => Stream<B>) => (s: Stream<A>) => Stream<B>
 
-declare export function continueWith <A> (f: (any) => A, s: Stream<A>): Stream<A>
-declare export function continueWith <A> (f: (any) => A): (s: Stream<A>) => Stream<A>
+declare export function continueWith <A> (f: (any) => String<A>, s: Stream<A>): Stream<A>
```

Comment:
Are you sure you want `string` here?

Reply:
Yea, what is this?

Reply:
Lol. Just making sure you're paying attention 😂

Reply:
Fixed in 0f4998e